.github/workflows/pre_release_version_bump.yml

```diff
@@ -0,0 +1,38 @@
+name: Manually Triggered Version Bumps
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "pre_release_version_bump"
+  pre_release_version_bump:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+        
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pennylane
+      - name: Run version bump
+        run: |
+          python vb_script.py --version_path "./_version.py" --changelog_path "./CHANGELOG.md" --pre_release
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
```

Comment:
Yes! @josh146, could we override and push this? I don't think its breaking warning, and I can't seem to find the white space being addressed. It could just be a bug on the code factor side ?

Reply:
done!